study.md

```diff
@@ -13,6 +13,10 @@ please do the following and be prepared to share and discuss during our next
 class.
 
 Submit detailed answers to these in this file via a pull request.
+You may want to submit pictures for your wireframes and/or user stories.
+[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+has instructions to link to a picture you've uploaded to a service like [Imgur]
+(http://imgur.com/).
```

Comment:
Seems more complicated to me. (And uses a bunch of vocab they might not know yet. But if you think it's a better way, I'm open to it.